SimG4CMS/PPS/interface/PPSStripOrganization.h

```diff
@@ -0,0 +1,19 @@
+#ifndef PPS_PPSStripOrganization_h
+#define PPS_PPSStripOrganization_h
+
+#include "globals.hh"
+#include "SimG4CMS/PPS/interface/TotemRPVDetectorOrganization.h"
+#include "G4Step.hh"
+
+class PPSStripOrganization : public TotemRPVDetectorOrganization {
+public:
+  inline PPSStripOrganization();
+  ~PPSStripOrganization() override = default;
+
+  uint32_t GetUnitID(const G4Step* aStep);
```

Comment:
@mundim , here as well 2 methods are not needed.

Reply:
Done.